ports/esp32/modesp32.c

```diff
@@ -192,6 +192,47 @@ STATIC mp_obj_t esp32_idf_heap_info(const mp_obj_t cap_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(esp32_idf_heap_info_obj, esp32_idf_heap_info);
 
+#if configUSE_TRACE_FACILITY
```

Comment:
Shouldn't this be `CONFIG_FREERTOS_USE_TRACE_FACILITY`?  That's how other sdkconfig params are referenced in the code.

Reply:
Ah, yes, that would fit better with the style for the rest of the code.

For reference, the lowercase `config` prefix is how the constants are defined in FreeRTOS itself; which are set by the ESP-IDF using it's own `CONFIG_FREERTOS_` definitions.